.github/workflows/linux_build.yml

```diff
@@ -65,9 +65,9 @@ jobs:
                 -DLINK_WITH_PYTHON=OFF \
                 -DUSE_PSYCHROMETRICS_CACHING=OFF \
                 -DUSE_GLYCOL_CACHING=OFF \
-                -OPENGL_REQUIRED=OFF \
-                -USE_PSYCH_STATS=ON \
-                -USE_PSYCH_ERRORS=OFF \
+                -DOPENGL_REQUIRED=OFF \
```

Comment:
I'm not sure how this worked before, but this should be the right way to do it.

Reply:
@mitchute without these two USE_PSYCH* lines, is it still exercising the code that depends on those two variables?

Reply:
@JasonGlazer no, those variables are not used without those CMake flags.